### PR TITLE
Reintegrate web WSL2 setup, change python from 3.7.7 to 3.8.5 ✨

### PR DESCRIPTION
**Pull request by gmanchon**
Features :
- reintegrate WSL2 setup from fullstack setup
- change python from 3.7.7 to 3.8.5
- add docker start/stop for ubuntu
- add ssh-agent usage

### CODE REVIEW

File: build.rb [hunk 1]
```diff
@@ -25,18 +26,26 @@
...
+  ubuntu_python
+  osx_virtualenv
   pip
-  win_sublime_package
-  win_make
-  win_extra
-  win_docker
+  ubuntu_docker
+  setup/ubuntu_inotify
+  setup/ubuntu_extra
```
Comment (barangerbenjamin) [inline build.rb:48]: Windows setup does not require the codec. Students will open the video on their browser installed on Windows directly, not through WSL 👍

**Review by barangerbenjamin — CHANGES_REQUESTED** [review]
Hey @gmanchon Solid work 💪 

Just a tiny comment on Windows Extra and I'm happy to merge it 🙌

File: build.rb (hunk 1 above)
Comment (gmanchon) [reply build.rb]: Happy to oblige 👍

**Comment by krokrob** [comment]
@barangerbenjamin I think we are OK there! Can you review/merge it?

**Comment by barangerbenjamin** [comment]
Yes absolutely! I meant to merge last week and got swamped!
Will re-read one last time and merge it 👍

**Review by barangerbenjamin — APPROVED** [review]
Let's go 🚀

**Pull request merged**